src/components/MintDialog/elements/ViewOnExplorer.tsx

```diff
@@ -10,33 +10,47 @@ interface ViewOnExplorerProps {
   onClick?: React.MouseEventHandler<HTMLButtonElement>
   isl1?: boolean
   title?: React.ReactNode
+  showIcon?: boolean
```

Comment:
what's this?

Reply:
The design removes the `RightArrow` icon if the interact with nft link is present